tests/js/searchtools.spec.js

```diff
@@ -95,7 +95,42 @@ describe('Basic html theme search', function() {
       ]];
       expect(Search.performTermsSearch(searchterms, excluded, terms, titleterms)).toEqual(hits);
     });
+  });
 
+  describe('unicode normalization', function() {
+    it('should find documents indexed with half-width characters using a full-width query', function() {
+      eval(loadFixture("normalization/searchindex.js"));
+
+      const orig = DOCUMENTATION_OPTIONS.SEARCH_UNICODE_NORMALIZATION;
+      try {
+        DOCUMENTATION_OPTIONS.SEARCH_UNICODE_NORMALIZATION = 'NFKC';
+        [_searchQuery, searchterms, excluded, ..._remainingItems] = Search._parseQuery('Ｓｐｈｉｎｘ');
+
+        terms = Search._index.terms;
+        titleterms = Search._index.titleterms;
+
+        hits = [[
+            "index",
+            "Main Page",
+            "",
+            null,
+            5,
+            "index.rst",
+            "text"],[
+            "sphinx",
+            "Sphinx",
+            "",
+            null,
+            15,
+            "sphinx.rst",
```

Comment:
```suggestion
            "simple_tokens.rst",
            "text"],[
            "sphinx",
            "Sphinx",
            "",
            null,
            15,
            "normalized_tokens.rst",
```

The reason for this suggestion is to communicate the purpose of each file more clearly to the reader of the test case.

Reply:
Thanks. I guess there's no reason to use the multiple files in this case.
I removed sphinx.rst and python.rst in 6fb1fb7c4.

Reply:
That makes sense - thank you!